src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Introductory NRQL tutorial: unlock your data"
+tags:
+  - NRQL tutorial
+  - 'NRQL: New Relic query language'
+  - Get started
+  - Tutorial
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to use NRQL with this basic, introductory tutorial."
+redirects:
+---
+
+We've covered the basics of NRQL in our other topics, such as what NRQL is, how NRQL works, and how NRQL relates to charts and dashboards. With this information, beginning to work with NRQL is the next step, and this tutorial is designed to help you do just that! Below we'll cover some foundational concepts within the language and get you set up with creating a some basic NRQL queries using your data. This tutorial will cover:
```

Comment:
- I wanted to have a sense of progression through the docs using the previous topics we've created/revamped and call out that readers may want to read them before starting the tutorial, but I'm unsure about the wording fitting with our language conventions. Any thoughts would be appreciated!

Reply:
you might have fixed it, didn't check, but typo of "creating a some basic..."